src/index.js

```diff
@@ -38,7 +38,12 @@ polyfillIntersectionObserver().then(() => {
     );
 
     renderElement(
-      <StoreProvider themes={themeNames} widths={widths}>
+      <StoreProvider
+        themes={themeNames}
+        widths={widths}
+        defaultVisibleThemes={playroomConfig.defaultVisibleThemes}
+        defaultVisibleWidths={playroomConfig.defaultVisibleWidths}
```

Comment:
Is there a reason to pass the config through to the provider from here? I ask because we already [import the config] in the store itself to setup some initial state.

[import the config]: https://github.com/seek-oss/playroom/blob/e2bd10e0f7f0d0509e352ebb4d0ad354c73f44f3/src/StoreContext/StoreContext.tsx#L19

Reply:
Ah I missed that playroom config was imported in the StoreContext. I've adjusted the logic to consume it there rather than passing it in here.